### PR TITLE
Database service controller

### PR DESCRIPTION
**Pull request by ghost**
Finished DatabaseServiceController

### CODE REVIEW

**Review by michibatoryu — APPROVED** [review]
sure thing!

File: src/main/java/edu/wpi/teame/controllers/FlowerRequestController.java [hunk 1]
```diff
@@ -35,68 +34,66 @@ public class FlowerRequestController implements IRequestController {
...
+
+    // Return to the home screen
     Navigation.navigate(Screen.HOME);
-    System.out.print(
-        "\nDelivery Type: "
-            + flowerRequestData.getRequestType()
-            + "\nRequest Data: "
-            + flowerRequestData.getRequestData());
+
+    DatabaseController db = new DatabaseController();
```
Comment (KalticCodes) [inline src/main/java/edu/wpi/teame/controllers/FlowerRequestController.java:77]: Use DatabaseController.INSTANCE

File: src/main/java/edu/wpi/teame/controllers/MealRequestController.java [hunk 2]
```diff
@@ -40,61 +41,54 @@ public void initialize() {
...
+
+    // Return to home screen
     Navigation.navigate(Screen.HOME);
-    System.out.print(
-        "\nDelivery Type: "
-            + mealRequestData.getRequestType()
-            + "\nRequest Data: "
-            + mealRequestData.getRequestData());
+
+    DatabaseController db = new DatabaseController();
```
Comment (KalticCodes) [inline src/main/java/edu/wpi/teame/controllers/MealRequestController.java:73]: Here too

**Review by KalticCodes — APPROVED** [review]
Everything seems good; changes approved

**Pull request merged**